### DISCUSSION TITLE
Please add a search function

### DISCUSSION THREAD

**Discussion by tritmox** [iSoron/uhabits#1809] [Feature Requests]
I track hundreds of metrics/ tasks/ habits.

**Comment by adrienbeau** [comment]
I am curious as to what kind of things these are. How can you have hundreds of habits? Are you not using the app as a kind of to-do list?

**Reply by turkerozturk** [reply]
I have around 100 habit entries. A search box filter is necessary. I think coding it is easy. SELECT * FROM tablename WHERE habitnames LIKE searchword

Some of my habits are atomic daily.
Some are not habits, they are to track my periodic payments.

Day by day I am adding new habit entries.

Some were already my routines.

What my target is, grouping öy habits.
Then leave or delegate some of them.

Because, do you know Maslow's requirements pyramid? In the end i wanna increase my valuable behaviours as habits.

Currently i am adding group labels as prefix to habit names.
